tests/test_binary_classification.py

```diff
@@ -24,42 +25,71 @@
 
 BINARY_CLASSIFICATION_SMALL = pl.DataFrame(
     {
-        "subject_id": [1, 2, 3],
+        "subject_id": np.arange(5).repeat(2),
```

Comment:
Why does the same subject_id have two different predictions for the same exact time? 

(This should not be a valid MEDS schema I think, if it currently is, and if it's not implemented yet we should implement a schema validator that checks for this)